Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TextToolbar/TextToolbarCode.bind

```diff
@@ -0,0 +1,54 @@
+private void Previewer_LinkClicked(object sender, LinkClickedEventArgs e)
+{
+    try
+    {
+        var linkOpen = Task.Run(() => Launcher.LaunchUriAsync(new Uri(e.Link)));
+    }
+    catch
+    {
```

Comment:
I'm pretty sure this will not be reachable. Check out : https://msdn.microsoft.com/en-us/magazine/jj991977.aspx